### ISSUE TITLE
Issue with reactive Vue js inside a tip

### ISSUE THREAD

**Issue by stratboy** [atomiks/tippyjs#730]
Hi, this is another tasty  issue :) So I've got some Tippy tips, which contains Vue reactive code. Too much code to show all, but anyway something like this (I think it's not so important right now):
```

                      <li v-if="screw.type === 'Altro'">
                        <ul>
                          <li v-if="screw.files"><b><?php _e('File vite caricati', 'fiam'); ?>:</b> {{ screw.files }}</li>
                          <li v-if="screw.file_sharing"><b><?php _e('Spedirò dei file di campione attraverso piattaforma di condivisione', 'fiam') ?>:</b> <?php _e('si', 'fiam'); ?></li>
                        </ul>
                      </li>
                      <li v-else>
                        <ul>
                          <li><b><?php _e('Tipo vite', 'fiam'); ?>:</b> {{ $t("screw_t.type['" + lwrc(screw.type) + "']") }}</li>
                          <li v-if="screw_dimensions(index)"><b><?php _e('Dimensioni', 'fiam'); ?> (mm):</b> <i>&nbsp;{{ screw_dimensions(index) }}</i></li>
                          <li v-if="screw.washer_type"><b><?php _e('Tipo di rondella', 'fiam'); ?>:</b> {{ $t("screw_t.washer_type['" + lwrc(screw.washer_type) + "']") }}</li>
                        </ul>
                      </li>
```

Ok. My issue is the following:

- The first time the tip gets shown, it shows correct data. 

- Then I change the data (it's a big multistep form where the user can go back and change some fields whose values are then show on the tips in following steps)

- The new data is reflected anywhere except for inside the tips. Therefore, there must be something that prevents the tip contents from being updated by Vue.

Any idea on how to solve?

Current Tippy code is the following:

```
    const component_screws_tips = delegate('.component-modules-container', {
      target: '.component-show-screw-details'
      ,trigger: 'click'
      ,allowHTML: true
      ,interactive: true
      ,content(button){
        let c = $(button).closest('.component-screw-box').find('.component-screw-details-tip');
        if(c[0] !== undefined){ return c[0].innerHTML; }
      }
    });
```

**Comment by atomiks** [comment]
`content` is only called once, even as a function. You need to call `.setContent(newContent)` whenever  the component updates.

**Comment by stratboy** [comment]
It's a pity. So a feature request could be: have the ability to use a dom element as the content, and have it re-evaluated each time it's showing up.

I tried to use the onShown event handler, like this:

```
      ,onShown: function(button){
        let c = $(button).closest('.component-screw-box').find('.component-screw-details-tip');
        console.log($(button)); // >>> correctly see the dom
        console.log($(button).parent()); // >>> empty

        if(c[0] !== undefined){
           component_screws_tips.setContent(c[0].innerHTML); doesn't work, c is undefined
        }
      }
```

But the problem is that in this case, unlike in the content property, the trigget (button) is passed apparently without any parent. I tried several ways to traverse up the dom tree, but it seems it's not possible from the onShown event.

So currently I'm forced to create my own tooltip code, after all, all I have to do is to show and hide an element, done tons of times. But quite a pity, having such a big library like tippy on the backline.
